meta.yml

```
@@ -0,0 +1,154 @@
+organization: math-comp
+opam_name: coq-mathcomp-analysis
+community: false
+action: true
+coqdoc: false
+
+synopsis: >-
+  An analysis library for mathematical components
+description: |-
+  This repository contains an experimental library for real analysis for
+  the Coq proof-assistant and using the Mathematical Components library.
+chat:
+- url: https://coq.zulipchat.com/login/#narrow/stream/237666-math-comp-analysis
+  shield: zulip
+authors:
+- name: Reynald Affeldt
+  initial: true
+- name: Cyril Cohen
+  initial: true
+- name: Marie Kerjean
+  initial: false
+- name: Assia Mahboubi
+  initial: true
+- name: Damien Rouhling
+  initial: true
+- name: Kazuhiko Sakaguchi
+  initial: false
+- name: Pierre-Yves Strub
+  initial: true
+
+opam-file-maintainer: Reynald Affeldt <reynald.affeldt@aist.go.jp>
+
+opam-file-version: dev
+
+license:
+  fullname: CeCILL-C
+  identifier: CECILL-C
+  file: LICENSE
+
+supported_coq_versions:
+  text: Coq 8.11 to 8.12
+  opam: '{ (>= "8.11" & < "8.13~") | (= "dev") }'
+
+tested_coq_opam_versions:
+- version: '1.11.0-coq-8.11'
+  repo: 'mathcomp/mathcomp'
+- version: '1.11.0-coq-8.12'
+  repo: 'mathcomp/mathcomp'
+
+dependencies:
+- opam:
+    name: coq-mathcomp-ssreflect
+    version: '{ (>= "1.11.0" & < "1.12~") }'
+  description: |-
+    [MathComp ssreflect 1.11](https://math-comp.github.io)
+- opam:
+    name: coq-mathcomp-fingroup
+    version: '{ (>= "1.11.0" & < "1.12~") }'
+  description: |-
+    [MathComp fingroup 1.11](https://math-comp.github.io)
+- opam:
+    name: coq-mathcomp-algebra
+    version: '{ (>= "1.11.0" & < "1.12~") }'
+  description: |-
+    [MathComp algebra 1.11](https://math-comp.github.io)
+- opam:
+    name: coq-mathcomp-solvable
+    version: '{ (>= "1.11.0" & < "1.12~") }'
+  description: |-
+    [MathComp solvable 1.11](https://math-comp.github.io)
+- opam:
+    name: coq-mathcomp-field
+    version: '{ (>= "1.11.0" & < "1.12~") }'
+  description: |-
+    [MathComp field 1.11](https://math-comp.github.io)
+- opam:
+    name: coq-mathcomp-finmap
+    version: '{ (>= "1.5.0" & < "1.6~") }'
+  description: |-
+    [MathComp finmap 1.5](https://github.com/math-comp/finmap)
+- opam:
+    name: coq-hierarchy-builder
+    version: '{ (>= "0.10.0" & < "0.11~") }'
+  description: |-
+    [Hierarchy Builder 0.10](https://github.com/math-comp/hierarchy-builder)
+namespace: mathcomp.analysis
+
+keywords:
+- name: analysis
+- name: topology
+- name: real numbers
+
+publications:
+- pub_url: https://jfr.unibo.it/article/view/8124
+  pub_title:  Formalization Techniques for Asymptotic Reasoning in Classical Analysis
+  pub_doi: 10.6092/issn.1972-5787/8124 
+- pub_url: https://hal.inria.fr/hal-02463336
+  pub_title: Competing inheritance paths in dependent type theory---a case study in functional analysis
+  pub_doi: 10.1007/978-3-030-51054-1_1
+- pub_url: http://www-sop.inria.fr/members/Damien.Rouhling/data/phd/thesis.pdf
+  pub_title: Outils pour la Formalisation en Analyse Classique
```

@affeldt-aist 
```suggestion
  pub_title: Formalisation Tools for Classical Analysis
```
@drouhling's thesis is in english, let's not deter our english readers by putting the french "by law" title, and let's not give false hopes to french speakers, because they won't get far in french only :laughing: